### PR TITLE
Added corazon_runner

### PR DESCRIPTION
**Pull request by jbcurtin**
Added corazon_runner to top-level of repository. Use it to run corazon codes. Check the readme for quick instructions. ( Note: Some work needs to be done to make it run-able on macOS with rsync )

### CODE REVIEW

**Review by mustaric — APPROVED** [review]
Joe this looks fine.  Thanks for pushing it into corazon.  If  you have time, you  could write a few words on what the utils functions actually do, that would be helpful, but not required.  Otherwise it looks good. Go ahead and merge.

File: corazon_runner/utils.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import os
+import subprocess
+import time
+import types
+
+from corazon_runner.constants import LOCAL_DATA_PATH
+from corazon_runner.datatypes import TESSLightCurveFile
+
+
+def locate_and_resolve_tess_datums(input_dir: str) -> types.GeneratorType:
```
Comment (jbcurtin) [inline corazon_runner/utils.py:11]: This function searches the local file-system for filenames that match expected input. ( spoc & qlp files ? )

After locating files that exist, the files are then returned out of the function as a TESSLightCurveFile datatype.

File: corazon_runner/utils.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    import os
+    import shutil
+    import tempfile
+
+    from corazon import run_pipeline
+
+    from tess_stars2px import tess_stars2px_function_entry
+
+    entries = []
+    for entry in locate_and_resolve_tess_datums(input_dir):
```
Comment (jbcurtin) [inline corazon_runner/utils.py:63]: Loads and counts entries found in various filenames (spoc & qlp) on disk

File: corazon_runner/utils.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    for idx, entry in enumerate(entries):
+        if idx % 100 == 0:
+            print(f'Chunk: {idx}')
+
+        tic_folder = os.path.join(output_dir, entry.output_dir, str(entry.tic))
+        if not os.path.exists(tic_folder):
+            os.makedirs(tic_folder)
+
+        # tic_folder = os.path.join(output_dir, str(entry.tic))
+        run_pipeline.run_write_one(entry.tic, entry.sector, tic_folder, entry.option, entry.local_dir)
```
Comment (jbcurtin) [inline corazon_runner/utils.py:83]: invokes pipeline code

**Pull request merged**